src/Compilers/VisualBasic/Test/Syntax/Parser/ParseMethods.vb

```diff
@@ -9,9 +9,11 @@ Imports Roslyn.Test.Utilities
 Public Class ParseMethods
     Inherits BasicTestBase
 
+    Private MyParseOptions As VisualBasicParseOptions = VisualBasicParseOptions.Default.WithImplicitDefaultOptionalParameter
+
     <WorkItem(917272, "DevDiv/Personal")>
     <Fact>
-    Public Sub ParseSub()
+    Public Sub ParseSub_A()
```

Comment:
@AlekseyTs You're correct, and I've updated the test to explicitly use the parse option required.